starter-apps/vue/vue-template/src/components/Header.vue

```diff
@@ -0,0 +1,92 @@
+<script setup>
+import { useI18n } from 'vue-i18n'
+import { useRouter } from 'vue-router'
+import { computed } from 'vue'
+
+import AppLink from '@/components/AppLink.vue'
+import { getLocalizedPath } from '@/router/index.js'
+import { getOtherLocale } from '@/i18n/index.js'
+import HeaderBreadcrumbs from '@/components/HeaderBreadcrumbs.vue'
```

Comment:
Could we rename this component from `HeaderBreadcrumbs` to simply `Breadcrumbs`? Since breadcrumbs are intended to be used only in the header, including "header" in the component name seems redundant.

Reply:
This is the typical naming convention in vue - I've had to add single-word components as exceptions in the eslint file so it ignores them. It is preferred to name things this way especially if they are nested within that component, in this case, `Breadcrumbs` are only found inside of `Header`, hence `HeaderBreadcrumbs`.

More info here: https://vuejs.org/style-guide/rules-strongly-recommended.html#base-component-names